hazardous/_ipcw.py

```diff
@@ -3,11 +3,134 @@
 from scipy.interpolate import interp1d
 from sklearn.base import clone
 from sklearn.ensemble import HistGradientBoostingClassifier
+from sklearn.preprocessing import MinMaxScaler
 from sklearn.utils.validation import check_is_fitted
 
 from .utils import check_y_survival
 
 
+class KaplanMeierEstimator:
```

Comment:
I 100% agree to have this at some stage, but after a discussion with @GaelVaroquaux, we thought that we needed to wait slightly before creating this wrapper, as it could make the wrong message to the community. Does this idea still hold, @GaelVaroquaux?